hal/halx86/CMakeLists.txt

```
@@ -66,7 +66,8 @@ if(ARCH STREQUAL "i386")
     add_hal(halxbox SOURCES xbox/halxbox.rc COMPONENTS xbox up)
     add_hal(halpc98 SOURCES pc98/halpc98.rc COMPONENTS pc98 up)
 
-    #add_hal(halmacpi SOURCES acpi/halacpi.rc COMPONENTS generic acpi smp pic)
+    #add_hal(halmacpi SOURCES smp/halmacpi.rc COMPONENTS generic acpi smp apic)
+    #add_hal(halmp SOURCES mp/halmp.rc COMPONENTS generic acpi smp apic)
```

```suggestion
    #add_hal(halmp SOURCES mp/halmp.rc COMPONENTS generic legacy smp apic)
```
MPS is legacy, not ACPI :)